app/src/main/java/ch/epfl/favo/user/User.java

```diff
@@ -53,17 +67,22 @@ public int getActiveAcceptingFavors() {
     return activeAcceptingFavors;
   }
 
-  void setActiveAcceptingFavors(int activeAcceptingFavors) {
-    this.activeAcceptingFavors = activeAcceptingFavors;
-  }
-
   public int getActiveRequestingFavors() {
     return activeRequestingFavors;
   }
 
+  void setActiveAcceptingFavors(int activeAcceptingFavors) {
+    this.activeAcceptingFavors = activeAcceptingFavors;
+  }
+
   void setActiveRequestingFavors(int activeRequestingFavors) {
     this.activeRequestingFavors = activeRequestingFavors;
   }
+
+  void setNotificationId(String notificationId) {
+    this.notificationId =  notificationId;
```

Comment:
Where do you set the notificationId?

Reply:
that is in the next PR, will be setting it in the SignInActivity.Java as we spoke about